docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+db:
+  image: mysql
+  expose:
+    - "3306"
+  ports:
+    - "3307:3306"
```

Comment:
Why do you expose this port to the host? And if you do, why use a different port than the default 3306?